learners/setup.md

```diff
@@ -2,25 +2,77 @@
 title: Setup
 ---
 
-## Pre-workshop Survey
+## How to get help
 
-Please remember to fill out the pre-workshop survey prior
-to the start of the workshop.
+- Attend a Git and GitHub Surgery.
+- Email the [support mailbox](mailto:ScienceGitMigrationProjectSupport@metoffice.gov.uk).
+
+Details of surgeries can be found on the
+Science Git Migration Project Comms Site.
+
+## Lesson Outcomes
+
+This lesson is split into two parts.
+Episodes in the first half before the break focus on Git.
+Episodes in the second half after the break focus on GitHub.
+
+By the end of the Git section you will be able to:
+
+- describe the importance of Version Control.
+- describe the similarities and differences between SVN and Git.
+- configure Git.
+- initialise new repositories using Git.
+- develop changes on a branch.
+- view differences between changes and explore your repositories history.
+
+By the end of the GitHub section you will be able to:
+
+- set up important GitHub settings and SSH access.
+- describe the similarities and differences between Trac and GitHub.
+- explain the differences between private, internal and public repositories.
+- set up a repository, including how to protect the `main` branch and enable
+  wiki pages, discussions, and GitHub projects.
+- view a repositories history and differences between changes.
+- view Issues and Pull Requests.
+- contribute changes to a repository using a Pull Request.
+
+And much more!
+
+## Pre-lesson Survey
+
+Please remember to fill out the
+[pre-lesson survey](https://forms.office.com/e/XJPJUTn0mp)
+prior to the start of the lesson.
 This information is vital for us to keep improving the lesson
 for other learners.
 
 ## Installing Git
 
-Since several Carpentries lessons rely on Git, please see
-[this section of the workshop template][workshop-setup] for
-instructions on installing Git for various operating systems.
+::: tab
+
+### Met Office Linux
+
+No setup is required, Git is installed for you.
+
+```bash
+$ git --version
+git version 2.47.0
```

Comment:
My version of `git` is different:

```
$ git --version
git version 2.36.6
```
Should we add a comment here to say "the version number may be different"?